src/icons/InfoSeal.tsx

```diff
@@ -0,0 +1,119 @@
+export default function ({ sadSeal }: { sadSeal?: boolean }) {
+  return sadSeal ? (
+    <svg
+      width="78"
+      height="65"
+      viewBox="0 0 78 65"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M3.70591 57.9185L35.5842 5.60535C37.143 3.04733 40.857 3.04734 42.4158 5.60536L74.2941 57.9185C75.9184 60.5841 73.9998 64 70.8783 64H7.12167C4.00021 64 2.0816 60.584 3.70591 57.9185Z"
+        fill="#1A0259"
+        stroke="#CF6AFF"
+        stroke-width="2"
+        stroke-linecap="round"
+      />
+      <path
+        d="M52.7472 35.458C51.9764 38.3346 49.0889 40.0603 46.2979 39.3124C43.5068 38.5646 41.8691 35.6263 42.6398 32.7497C43.4106 29.8731 44.9025 33.3559 47.6935 34.1038C50.4846 34.8516 53.518 32.5813 52.7472 35.458Z"
+        stroke="#EFECD6"
+        stroke-width="2"
+        stroke-linecap="round"
```

Comment:
Remove all `stroke-linecap="round"`, they're set by default, this is a dead code

Reply:
fixed, ty